actionpack/CHANGELOG.md

```diff
@@ -18,6 +18,24 @@
 
     *Greg Ose & Ben Toews*
 
+*   Add `ActionController::Renderers::SERIALIZERS` to separate rendering from serialization.
```

Comment:
I'm wondering if `adapters` or `formatters` might be a better name here than `serializers`